hack/logging-dump.sh

```diff
@@ -164,22 +162,12 @@ check_fluentd_connectivity() {
   oc exec $pod -- curl -ILvs --key /etc/fluent/keys/key --cert /etc/fluent/keys/cert --cacert /etc/fluent/keys/ca -XGET https://$es_host:$es_port &>> $fluentd_folder/$pod
   echo "  without ca" >> $fluentd_folder/$pod
   oc exec $pod -- curl -ILkvs --key /etc/fluent/keys/key --cert /etc/fluent/keys/cert -XGET https://$es_host:$es_port &>> $fluentd_folder/$pod
-  echo --Connectivity between $pod and elasticsearch-ops >> $fluentd_folder/$pod
-  es_host=$(oc get pod $pod  -o jsonpath='{.spec.containers[0].env[?(@.name=="OPS_HOST")].value}')
-  es_port=$(oc get pod $pod  -o jsonpath='{.spec.containers[0].env[?(@.name=="OPS_PORT")].value}')
-  if [ -n "$es_host" -a -n "$es_port" ] ; then
-    echo "ops cluster Elasticsearch"
-    echo "  with ca" >> $fluentd_folder/$pod
-    oc exec $pod -- curl -ILvs --key /etc/fluent/keys/key --cert /etc/fluent/keys/cert --cacert /etc/fluent/keys/ca -XGET https://$es_host:$es_port &>> $fluentd_folder/$pod
-    echo "  without ca" >> $fluentd_folder/$pod
-    oc exec $pod -- curl -ILkvs --key /etc/fluent/keys/key --cert /etc/fluent/keys/cert -XGET https://$es_host:$es_port &>> $fluentd_folder/$pod
-  fi
 }
 
 check_fluentd_persistence() {
```

Comment:
need a check_rsyslog_persistence - or perhaps a `check_collector_persistence` that takes `fluentd` or `rsyslog` (or `promtail`) as a parameter